src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Machine {
```

Comment:
According to the checklist, class elements should have appropriate access modifiers. Consider if the abstract class `Machine` should be public to be accessible by other classes in different packages.